### ISSUE TITLE
Documentation for sort

### ISSUE THREAD

**Issue by dvd101x** [josdejong/mathjs#2892]
Hi, I found a minor issue while parsing `help(sort)`

The issue is one of the examples shows an error

```
sort(["C", "B", "A", "D"])
        Error: Cannot convert "C" to a number
```
By reading the documentation it seems for it to work it should be

```
sort(["C", "B", "A", "D"], compareNatural)
        ["A", "B", "C", "D"]
```

**To Reproduce**
Evaluate `help(sort)` on the demo

**Comment by josdejong** [comment]
Thanks for reporting David, keep them coming :thumbsup:. Fixed via 3a77a6f (not yet published)

**Comment by dvd101x** [comment]
Thank you Jos!

**Comment by josdejong** [comment]
Fixed definitions are now published in `v11.6.0`.
